app-backend/api/src/main/scala/uploads/package.scala

```diff
@@ -12,7 +12,7 @@ package object uploads {
             .filter { p =>
                 val _p = p.toLowerCase
                 _p.endsWith(".tif") ||
-                _p.endsWith(".tif")
+                _p.endsWith(".tiff")
```

Comment:
I don't think we can just straight add them -- as it stands, we [pull in the entire list](https://github.com/raster-foundry/raster-foundry/blob/develop/app-backend/api/src/main/scala/uploads/Routes.scala#L72-L73) of files that we find, so adding extra formats carries a pretty serious cost in that `process_upload` wouldn't know what to do with them